frameworks/g-cloud-8/questions/declaration/dunsNumber.yml

```diff
@@ -0,0 +1,3 @@
+question: What is your organisation’s DUNs number?
```

Comment:
Everywhere else we have all capitals - i.e. `DUNS` rather than `DUNs`.  We should make it the same everywhere.